### PR TITLE
Update Vietnamese localization

### PR DESCRIPTION
**Pull request by vinhphm**
This PR enhances the Vietnamese localization with the following improvements:

- Adds new translation strings introduced by Bluesky
- Updates terminology to align with established Vietnamese translations from Twitter for better consistency
- Improves contextual accuracy of certain translations
- Fixes typos and grammar issues

Special thanks to the Bluesky team and contributors for the initial Vietnamese translations.

### CODE REVIEW

**Comment by EmbiiFox** [comment]
Hi there, I was initially preparing to update the Vietnamese translations myself when I discovered your pull request. As a result, I'd like to provide some feedback and review.
I agree with most of the changes you've made, particularly:
- Changing "người gắn nhãn" to "dịch vụ gắn nhãn" - this better captures Bluesky's intent of referring to labeling as a service rather than a specific person.
- Modifying "tắt thông báo" to "ẩn" in the "mute" context - this more accurately reflects the functionality of preventing posts from appearing, rather than simply disabling notifications.

I noticed that "Chat" remains untranslated. While this might be a bit unusual at first, leaving it the way it is could be indeed more concise and natural-sounding.

Overall, these are solid improvements to the Vietnamese localization. I approve the pull request.

**Review by EmbiiFox — COMMENTED** [review]
Hi there, I was initially preparing to update the Vietnamese translations myself when I discovered your pull request. As a result, I'd like to provide some feedback and review.
I agree with most of the changes you've made, particularly:
- Changing "người gắn nhãn" to "dịch vụ gắn nhãn" - this better captures Bluesky's intent of referring to labeling as a service rather than a specific person.
- Modifying "tắt thông báo" to "ẩn" in the "mute" context - this more accurately reflects the functionality of preventing posts from appearing, rather than simply disabling notifications.
I noticed that "Chat" remains untranslated. While this might be a bit unusual at first, leaving it the way it is could be indeed more concise and natural-sounding.
Overall, these are solid improvements to the Vietnamese localization. I approve the pull request.

**Comment by vinhphm** [comment]
Thank @EmbiiFox for spending time going through my changes. I'm glad we share some similar perspectives on the translations. And yes, I also think it's best to leave "chat" as it is, since "cuộc trò chuyện" would be a bit too wordy and I don't think anyone actually uses that in daily life 😄.

Hi @pfrazee, could we also have Vietnamese added as one of the supported languages in App Store? I don’t think it’s there yet.

IMG_4148

**Comment by surfdude29** [comment]
@vinhphm Hi, I have a PR open (#6835) which will add Vietnamese (and other languages recently added) to the App Store listing and will also translate the `TextInput` context menus for iOS users with their device set to Dutch or Vietnamese (they still appear in English at the moment).

Hopefully @pfrazee or one of the other team members will be able to have a look at it and merge it soon :)

**Comment by surfdude29** [comment]
#6835 has now been merged, so whenever the next release is cut and becomes available in the App Store, Vietnamese should then appear in the Languages list :)

**Review by pfrazee — APPROVED** [review]
Awesome, thanks all!
